_layouts/useCaseBase.html.haml

```diff
@@ -1,21 +1,14 @@
 ---
 layout: base
 ---
-- relatedPosts = site.posts.select{|e| !e.tags.nil? and e.tags.any?{|t| t.to_s.eql? page.related_tag} }
 .row
   .col-md-9
     ~ content
-    - if !relatedPosts.empty?
-      %h2 Related blog posts
-      %ul
-        - for post in relatedPosts
-          %li(style="margin-top: 10px;")
-            %a{:href => relative(post.url)}
-              = post.title
-            = partial('userBadge.html.haml', :outputPage => page, :userId => post.author, :mode => 'inline')
+    = partial('relatedBlogPosts.html.haml', :outputPage => page, :related_tag => page.related_tag)
   .col-md-3
     = partial('downloadJumbotron.html.haml', :outputPage => page)
     = partial('documentationJumbotron.html.haml', :outputPage => page)
-    = partial('latestBlogPosts.html.haml', :outputPage => page)
     = partial('latestReleases.html.haml', :outputPage => page)
     = partial('events.html.haml', :outputPage => page)
+    = partial('latestBlogPosts.html.haml', :outputPage => page)
+    = partial('latestVideos.html.haml', :outputPage => page)
```

Comment:
I wonder if we could reuse `_partials/whatIsOptaPlanner.html.haml` here because each use case document now has a duplicated paragraph called "What is OptaPlanner?" with more or less the same content.

Reply:
I want to work towards that. First the index page must reuse it, but that didn't work well and wanted to get this merged first - looking into that later.
For the use case pages it's more difficult because the first paragraph differs - but we can use a parameter for the partial to make that dynamic.